papis/commands/doctor.py

```diff
@@ -367,18 +367,40 @@ def key_type_check(doc: papis.document.Document) -> List[Error]:
     """
     Check document keys have expected types.
 
-    The ``doctor-key-type-check-keys`` configuration entry defines a mapping
-    of keys and their expected types.
+    The :ref:`config-settings-doctor-key-type-check-keys` configuration entry
+    defines a mapping of keys and their expected types. If the desired type is a
+    list, the :ref:`config-settings-doctor-key-type-check-separator` value can
+    be used to split an existing string.
```

Comment:
The :ref:`config-settings-doctor-key-type-check-keys` configuration entry defines a mapping of keys and their expected types. If the desired type is a list, the :ref:`config-settings-doctor-key-type-check-separator` value can be used to split an existing string (and, similarly, if the desired type is a string, it can be used to join a list of items).

Reply:
Added this too! At the moment the docs for these methods don't get rendered anywhere, but it's good to be clear.